test/unit/app/models/document/paginated_timeline_test.rb

```diff
@@ -17,7 +17,7 @@ class PaginatedTimelineTest < ActiveSupport::TestCase
       assert_equal entry_timestamps.sort.reverse, entry_timestamps
     end
 
-    it "is is a list of VersionDecorator and EditorialRemark objects when no 'only' argument is passed in" do
+    it "is is a list of VersionDecorator and EditorialRemark objects when no 'only' argument is not passed in" do
```

Comment:
This is difficult to parse 😅 can we reword to remove the double negative?

Similarly on line 92